packages/@okta/vuepress-site/docs/guides/sign-users-out/main/index.md

```diff
@@ -18,25 +18,22 @@ An important part of security is to minimize the chances that a malicious actor
 
 **What you need**
 
-* An Okta Developer Edition organization. Don't have one? [Create one for free](https://developer.okta.com/signup).
-* An app that can sign in to Okta. To create your own, see the following guides:
-  * [Sign users in to your web app](/docs/guides/sign-into-web-app/)
-  * [Sign users in to your mobile app](/docs/guides/sign-into-mobile-app/)
-  * [Sign users in to your single-page app](/docs/guides/sign-into-spa/)
+* [Okta Developer Edition organization](https://developer.okta.com/signup)
+* An app that can sign in to Okta
 
 **Sample code**
 
 <StackSelector snippet="samplecode" noSelector/>
 
 ---
 
-## Overview
+## About signing users out of an app
```

Comment:
Again, "About" reads a bit weird to me, although I don't feel that strongly about it. If this is the decided standard, and you've done it everywhere, then I'm not going to fight it.

Reply:
"About..." was decided as the standard by the dev docs team and has been updated as such.